src/test/java/edu/harvard/iq/dataverse/api/LicensesIT.java

```diff
@@ -144,6 +144,20 @@ public void testLicenses(){
         status = JsonPath.from(body).getString("status");
         assertEquals("OK", status);
         
+        //Fail trying to set null sort order
```

Comment:
I'm getting an error in a prior test - up in row 111 above where if I leave my existing licenses sort orders null, the get license by id which is expected to fail will succeed, because the license with id 1 gets sorted to the end of the array when id 2 is set to the default license.

Reply:
I have changed the code such that the `licenseId` is now the one with the highest ID found. It should fix this issue.

Reply:
Thanks!